lib/xebow/keys.ex

```diff
@@ -102,20 +119,65 @@ defmodule Xebow.Keys do
     poll_timer_ms = 15
     :timer.send_interval(poll_timer_ms, self(), :update_pin_values)
 
+    animations =
+      Animation.types()
```

Comment:
:information_source: (informational, no action required)

It would be nice to eventually move the runtime animation types into a file that handles application logic (`xebow.ex`).

Related: https://github.com/ElixirSeattle/xebow/issues/49#issuecomment-653718939